### DISCUSSION TITLE
.NET Framework 4.8 support

### DISCUSSION THREAD

**Discussion by qiangqiang101** [JohnnyCrazy/SpotifyAPI-NET#671] [General]
I still on VS 2015 and some other lib doesn't support for 5.0 yet so I'm stuck right now

**Comment by JohnnyCrazy** [comment]
The project supports .net standard 2.0, you should be able to use it in .net 4.8

**Reply by qiangqiang101** [reply]
Severity	Code	Description	Project	File	Line	Suppression State
Error		Could not install package 'SpotifyAPI.Web 6.2.1'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.7.2', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.			0

**Reply by JohnnyCrazy** [reply]
I just tested it with VS 2019, created new WinForms project based on .NET 4.7.2 and was able to install the library without problems.

Maybe try to update your NuGET client? I heard VS 2015 included NuGET may not work by default

**Reply by rysonw** [reply]
Any fix on this? I am getting the same error on VS 2022/.NET 4.7.2

**Reply by Leonard2033** [reply]
Any fix for this Problem ?
Same Error on WPF-App with .Net Framework 4.8

**Reply by farsdewibs0n** [reply]
Bump

**Reply by meetox80** [reply]
bump. 4.7.x/4.8 support

**Comment by JohnnyCrazy** [comment]
Should be solved by https://github.com/JohnnyCrazy/SpotifyAPI-NET/releases/tag/7.0.2
